### PR TITLE
Feature No Result

### PR DESCRIPTION
**Pull request by Juliana-Dimeglio**
Added a text and an image when the search has no results found 
Captura de pantalla 2021-09-15 172248

### CODE REVIEW

File: lib/pages/home_body_content.dart [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import 'package:flutter/material.dart';
-
+import 'package:tmdb_movies/utils/string_constants.dart';
```
Comment (MarcosCGlobant) [inline lib/pages/home_body_content.dart]: relative import here, please

File: lib/pages/home_page.dart [hunk 2]
```diff
@@ -42,90 +42,92 @@ class _HomeState extends State<Home> {
...
+        bottomNavigationBar: BottomNavigationBar(
+          backgroundColor: Colors.black,
+          unselectedItemColor: Colors.grey.shade400,
+          currentIndex: index,
+          items: [
+            BottomNavigationBarItem(
+              icon: Icon(
+                Icons.home,
+              ),
+              label: "Home",
```
Comment (MarcosCGlobant) [inline lib/pages/home_page.dart]: This value should be constant

File: lib/pages/home_page.dart [hunk 3]
```diff
@@ -42,90 +42,92 @@ class _HomeState extends State<Home> {
...
-            label: "Home",
-          ),
-          BottomNavigationBarItem(
-            icon: Icon(
-              Icons.person,
+            BottomNavigationBarItem(
+              icon: Icon(
+                Icons.person,
+              ),
+              label: "About",
```
Comment (MarcosCGlobant) [inline lib/pages/home_page.dart]: and this one as well

File: lib/pages/movie_details_page.dart [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import 'package:exported_movie_widgets/exported_movie_widgets.dart';
 import 'package:flutter/material.dart';
+import 'package:tmdb_movies/utils/measures_constants.dart';
```
Comment (MarcosCGlobant) [inline lib/pages/movie_details_page.dart]: relative import here, please

File: lib/pages/movie_details_page.dart [hunk 5]
```diff
@@ -12,32 +13,80 @@ class MovieDetailPage extends StatelessWidget {
...
+                  backgroundColor: MaterialStateProperty.all<Color>(
+                    Colors.black,
+                  ),
+                ),
+                icon: Icon(
+                  Icons.arrow_back,
+                  size: 40,
+                ),
+                label: Text(
+                  "",
```
Comment (MarcosCGlobant) [inline lib/pages/movie_details_page.dart]: this one can be an empty string constant

File: lib/pages/movie_details_page.dart [hunk 6]
```diff
@@ -12,32 +13,80 @@ class MovieDetailPage extends StatelessWidget {
...
+                imagePath: result.posterPath,
+                heroTag: StringConstants.heroMovieDetailTransitionTag +
+                    result.id.toString(),
+              ),
+              MovieDetailsTitle(
+                title: result.title,
+              ),
+              Padding(
+                padding: const EdgeInsets.symmetric(
+                  horizontal: 10,
```
Comment (MarcosCGlobant) [inline lib/pages/movie_details_page.dart]: this padding values can be constants as well

**Review by MarcosCGlobant — COMMENTED** [review]
Some code style changes

File: lib/pages/home_body_content.dart [hunk 7]
```diff
@@ -35,18 +35,42 @@ class _HomeBodyContentState extends State<HomeBodyContent> {
...
-            ? GridView.count(
-                mainAxisSpacing: widget.mainAxisSpacement,
-                childAspectRatio: widget.childAspectRatio,
-                crossAxisCount: widget.crossAxisCount,
-                children: buildList(
-                  context,
-                  snapshot,
-                ),
-              )
+            ? snapshot.data.results.length > 0
```
Comment (FranciscoGaraventa) [inline lib/pages/home_body_content.dart]: This can be snapshot.data.results.isNotEmpty?

File: lib/pages/home_body_content.dart [hunk 8]
```diff
@@ -35,18 +35,42 @@ class _HomeBodyContentState extends State<HomeBodyContent> {
...
+                ? GridView.count(
+                    mainAxisSpacing: widget.mainAxisSpacement,
+                    childAspectRatio: widget.childAspectRatio,
+                    crossAxisCount: widget.crossAxisCount,
+                    children: buildList(
+                      context,
+                      snapshot,
+                    ),
+                  )
+                : SingleChildScrollView(
```
Comment (FranciscoGaraventa) [inline lib/pages/home_body_content.dart:51]: This SingleChildScrollView is necessary?

File: lib/pages/home_page.dart [hunk 9]
```diff
@@ -42,90 +42,92 @@ class _HomeState extends State<Home> {
...
-        backgroundColor: Colors.black,
-        unselectedItemColor: Colors.grey.shade400,
-        currentIndex: index,
-        items: [
-          BottomNavigationBarItem(
-            icon: Icon(
-              Icons.home,
+        body: pages[index],
+        bottomNavigationBar: BottomNavigationBar(
+          backgroundColor: Colors.black,
```
Comment (FranciscoGaraventa) [inline lib/pages/home_page.dart]: Instead to use the same color that Scaffold has, i think you can use Theme.of(context)

File: lib/pages/movie_details_page.dart [hunk 10]
```diff
@@ -12,32 +13,80 @@ class MovieDetailPage extends StatelessWidget {
...
+              ElevatedButton.icon(
+                onPressed: () => Navigator.pop(context),
+                style: ButtonStyle(
+                  backgroundColor: MaterialStateProperty.all<Color>(
+                    Colors.black,
+                  ),
+                ),
+                icon: Icon(
+                  Icons.arrow_back,
+                  size: 40,
```
Comment (FranciscoGaraventa) [inline lib/pages/movie_details_page.dart]: This size could be added to constant file

File: lib/pages/movie_details_page.dart [hunk 11]
```diff
@@ -12,32 +13,80 @@ class MovieDetailPage extends StatelessWidget {
...
+                ),
+                label: Text(
+                  "",
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.only(
+                  top: MeasuresConstants.logoPaddingTop,
+                  left: 100.0,
```
Comment (FranciscoGaraventa) [inline lib/pages/movie_details_page.dart]: This can be added to a constant file too

File: lib/pages/movie_details_page.dart [hunk 12]
```diff
@@ -12,32 +13,80 @@ class MovieDetailPage extends StatelessWidget {
...
+                  fontSize: 17,
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.symmetric(
+                  horizontal: 5,
+                ),
+                child: MovieDetailsOverview(
+                  overview: result.overview,
+                  fontSize: 20,
```
Comment (FranciscoGaraventa) [inline lib/pages/movie_details_page.dart]: fontSize and horizontal can be added to a constant file

File: lib/pages/home_body_content.dart (hunk 1 above)
Comment (Juliana-Dimeglio) [reply lib/pages/home_body_content.dart]: Fixed

File: lib/pages/home_page.dart (hunk 2 above)
Comment (Juliana-Dimeglio) [reply lib/pages/home_page.dart]: Fixed

File: lib/pages/home_page.dart (hunk 3 above)
Comment (Juliana-Dimeglio) [reply lib/pages/home_page.dart]: Fixed

File: lib/pages/movie_details_page.dart (hunk 4 above)
Comment (Juliana-Dimeglio) [reply lib/pages/movie_details_page.dart]: Fixed

File: lib/pages/movie_details_page.dart (hunk 5 above)
Comment (Juliana-Dimeglio) [reply lib/pages/movie_details_page.dart]: Fixed

File: lib/pages/movie_details_page.dart (hunk 6 above)
Comment (Juliana-Dimeglio) [reply lib/pages/movie_details_page.dart]: Fixed

File: lib/pages/home_body_content.dart (hunk 7 above)
Comment (Juliana-Dimeglio) [reply lib/pages/home_body_content.dart]: Fixed

File: lib/pages/home_page.dart (hunk 9 above)
Comment (Juliana-Dimeglio) [reply lib/pages/home_page.dart]: Fixed

File: lib/pages/movie_details_page.dart (hunk 10 above)
Comment (Juliana-Dimeglio) [reply lib/pages/movie_details_page.dart]: Fixed

File: lib/pages/movie_details_page.dart (hunk 11 above)
Comment (Juliana-Dimeglio) [reply lib/pages/movie_details_page.dart]: Fixed

File: lib/pages/movie_details_page.dart (hunk 12 above)
Comment (Juliana-Dimeglio) [reply lib/pages/movie_details_page.dart]: Fixed

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Review by FranciscoGaraventa — APPROVED** [review]
LG

**Pull request merged**